src/App.js

```diff
@@ -6,49 +6,56 @@ class App extends React.Component {
     super(props);
     this.state = {
       searchQuery: '',
-      locationData: {}
+      showMapAndCityInfo: false
     }
   }
 
-  getLoco = async () => {
-    let API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_ACCESS_TOKEN}&q=${this.state.searchQuery}&format=json`;
-    let resp = await axios.get(API);
-    console.log(resp.data[0])
-    this.setState({ location:resp.data[0] });
-  }
+  handleInput = e => this.setState({ searchQuery: e.target.value });
 
+  getLoco = async e => {
+    e.preventDefault();
+    
 
+    let API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_ACCESS_TOKEN}&q=${this.state.searchQuery}&format=json`;
 
 
-  handleSubmit = e => {
-    e.preventDefault();
-
-    let searchQuery = e.target.city.value
+    let locoResults = await axios.get(API)
+    console.log(locoResults.data)
     this.setState({
-      searchQuery,
+      locoData: locoResults.data[0],
+      showMapAndCityInfo: true
     })
+  
   }
 
 
+
   render() {
-    console.log(this.state.searchQuery);
+    console.log(this.state)
     return (
       <>
         <header>
           <h1>City Explorer</h1>
         </header>
         <main>
-          <form onSubmit={this.handleSubmit}>
+          <form onSubmit={this.getLoco}>
             <label>Enter a City!
-              <input name="city" type="text" />
+              <input type="text" onInput={this.handleInput} />
             </label>
-            <button type="submit">Explore!</button>
+            <button>Explore!</button>
           </form>
-          <h2>The city is: {this.state.location.display_name}</h2>
+
+          {
+            this.state.showMapAndCityInfo &&
+            <article>
+              <h3>{this.state.locoData.display_name}</h3>
+              <img src= {`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_ACCESS_TOKEN}&zoom=10&center=${this.state.locoData.lat},${this.state.locoData.lon}`} alt="map results" />
```

Comment:
I am getting a 403 here so I am unable to view map or city data after submitting form. Can you:

- grab a TA and verify this url is correct 
- Ideally we extract this information out of `src` because it contains valuable information. We can use it elsewhere in a new method or perhaps even in the one you use to fetch city data. 👍